packages/react/src/components/table/table.tsx

```diff
@@ -214,7 +240,7 @@ export interface TableProps<T extends object> {
      * @default medium
      */
     rowSize?: RowSize;
-    selectableRows?: boolean;
+    selectionMode?: SelectionMode;
```

Comment:
On a un `expandableRows?: 'single' | 'multiple';`, pourquoi on ne garderait pas la meme nomenclature.

```suggestion
    selectableRows?: 'single' | 'multiple';
```

Reply:
C'est débatable. Honnnêtement je trouve que `selectionMode` est pas mal plus clair sur la fonction de la prop que `selectableRows` imho.

Reply:
Yeah, pour moi c'est pas tant le nom mais plus le fait qu'on garde une certaine similitude entre les noms. Ca pourrait etre expandableRows qui se renomme a `expandMode` aussi.

Reply:
Par contre je me demande si on ne devrait pas garder `row` ou `rows` dans les noms de props, vu que ça affecte les rows et non la table. `row(s)Selection` et `row(s)Expansion`?